frontend/packages/console-plugin-sdk/README.md

```diff
@@ -0,0 +1,234 @@
+# OpenShift Console Plugins
+
+Openshift Console Plugins extend the functionality of the console. Plugins are enabled by
+adding them as `dependencies` of the [console-app](frontend/packages/console-app) package.
+
+- Plugins are static, their code gets bundled ("baked") into Console build output:
+  - Plugins are not loaded dynamically at runtime, which in turn simplifies the architecture (no need to sandbox plugin execution or its interaction with Console app or Bridge server).
+  - Console-app dependencies control the set of plugins to include in Console build, this can be overridden via CONSOLE_PLUGINS env. variable.
+- Plugins have their code represented as Console monorepo packages:
+  - This increases the level of trust between Console app and its plugins.
+  - Plugin code close to Console code avoids cross-repo PR maintenance issues.
+  - Each plugin has an OWNERS file denoting its ownership (see here for details on reviewers and approvers).
```

Comment:
> Is "here" supposed to be a link?

It was, in my original comment :smiley: 

> Whenever I see something like "see here", I automatically assume the "here" is a link.

:+1: 

A good documentation favors specific terms, like "`OWNERS` file section of the Prow `approve` plugin", instead of loose terms, like "here" or "this link".